### ISSUE TITLE
Missing the regularization term in softmax training loss ?

### ISSUE THREAD

**Issue by kevinlemon** [fxia22/pointnet.pytorch#16]
Hello.
Your PointNet project is great. But in your code it seems that you missed the regularization term in softmax training loss, the equation (2) in the paper.
